alf/algorithms/data_transformer.py

```diff
@@ -600,11 +600,21 @@ def __init__(self,
         self._update_mode = update_mode
 
     def _transform(self, timestep_or_exp):
+        norm = self._normalizer
         if ((self._update_mode == "replay" and common.is_replay())
                 or (self._update_mode == "rollout" and common.is_rollout())):
-            self._normalizer.update(timestep_or_exp.reward)
+            norm.update(timestep_or_exp.reward)
+
+            with alf.summary.scope("RewardNormalizer"):
```

Comment:
This part should be a member function of Normalizer as it accesses private members.

Reply:
Deleted. Just realized that the normalizer already has a summary function inside

Reply:
> This part should be a member function of Normalizer as it accesses private members.

Deleted. Just realized that the normalizer already has a summary function inside

Reply:
Make sure you are aware that normalizer won't summarize if debug_summaries is not True

Reply:
> Make sure you are aware that normalizer won't summarize if debug_summaries is not True

Yeah I've passed a flag in the gin file